src/hydrate/platform/proxy-host-element.ts

```diff
@@ -131,6 +134,7 @@ export function proxyHostElement(elm: d.HostElement, cstr: d.ComponentConstructo
         });
       }
     });
+    (cstr as any).prototype.done = true;
```

Comment:
Should we make the property name more clear, e.g. `__stencilAugmented`? `done` sounds more like an official web interface.

Reply:
good suggestion - thanks.
I didn't like 'done' was just a placeholder. I think it could have caused name clashes too.

I was also trying to see if there's some better place to set this, but I think it is in-fact here... as it deals with setup / initial values etc.